test/fluree/db/query/fql_test.clj

```diff
@@ -222,3 +222,38 @@
                           :schema/name  "Brian"}]}
                subject)
             "returns all results in a map keyed by alias.")))))
+
+(deftest ^:integration subject-object-test
```

Comment:
Since this is meant to fix deletion in https://github.com/fluree/core/issues/18, could we also get a corresponding test in `update-test` ns?